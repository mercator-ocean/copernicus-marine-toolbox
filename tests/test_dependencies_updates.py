from toml import load


class TestDependeciesUpdates:
    def test_update_dependencies(self, snapshot):

        with open("pyproject.toml") as f:
            config = load(f)
        # If these change, update the dependencies in the documentation!
        assert config["tool"]["poetry"]["dependencies"] == snapshot
