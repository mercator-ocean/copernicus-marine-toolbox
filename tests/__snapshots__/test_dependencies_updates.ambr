# serializer version: 1
# name: TestDependeciesUpdates.test_update_dependencies
  dict({
    'arcosparse': '==0.1.4',
    'boto3': '>=1.26',
    'click': '>=8.0.4',
    'dask': '>=2022',
    'h5netcdf': '^1.4.0',
    'lxml': '>=4.9.0',
    'numpy': '>=1.23.0',
    'pydantic': '^2.9.1',
    'pystac': '>=1.8.3',
    'python': '>=3.9',
    'requests': '>=2.27.1',
    'semver': '>=3.0.2',
    'setuptools': '>=68.2.2',
    'tqdm': '>=4.65.0',
    'xarray': '>=2023.4.0',
    'zarr': '^2.13.3',
  })
# ---
